src/cpp/rtps/transport/test_UDPv4Transport.h

```diff
@@ -101,15 +101,15 @@ class test_UDPv4Transport : public UDPv4Transport
             const fastrtps::rtps::octet* buffer,
             uint32_t size);
     bool packet_should_drop(
-            const fastrtps::rtps::octet* send_buffer,
-            uint32_t send_buffer_size);
+            const std::array<asio::const_buffer, 3>& send_buffers,
+            uint32_t total_bytes);
```

Comment:
For example, here the name of the variable has been changed but this change has not been applied in the implementation.

Reply:
Changed in 33959d01d5afabd1b0a76f7df4238b632fca3b57